src/modules/todos.js

```diff
@@ -58,11 +105,35 @@ export const markToDo = (e, todos) => {
   renderToDoList(todos);
 };
 
+/* 
+const markToDo = (e, todos) => {
+  const clickedCheckbox = e.target.closest('.todo-checkbox');
+  const clickedToDo = clickedCheckbox. nextElementSibling;
+  const toDoIndex = todos.findIndex(
+    (toDoItem) => toDoItem.toDoItem === clickedToDo.value,
+  );
+  todos = getTodo();
+  todos[toDoIndex].completed = !todos[toDoIndex].completed;
+  setToDo(todos);
+  renderToDoList(todos);
+}
+*/
+
 // Clear Completed Todo
 export const clearToDos = (todos) => {
   todos = todos.filter((toDoItem) => toDoItem.completed === false);
   todos.forEach((toDoItem, index) => {
     toDoItem.id = index + 1;
   });
   return todos;
-};
\ No newline at end of file
+};
+
+/*
+const clearToDos = (todos) => {
+  todos = todos.filter((toDoItem) => toDoItem.completed === false);
+  todos.forEach((toDoItem, index) => {
+    toDoItem.id = index + 1;
+  });
+  return todos;
+}
+*/
```

Comment:
Please delete commented code inside your code because it makes your code unclean. 😉